chapter2/scenarios/06_Dwarven_Doors.cfg

```diff
@@ -513,15 +513,15 @@
     {NOTRAIT_UNIT 2 Swordsman 7 7}
     [message]
       type=Swordsman
-      message=_"Are you finished with the armors?"
+      message=_"Are you finished with the armour suits?"
     [/message]
     [message]
       type=Dwarvish Runesmith
       message=_"Yes, I am."
     [/message]
     [message]
       type=Swordsman
-      message=_"Ok let me see. (examining armors) They are very strong and nice."
+      message=_"Ok let me see. (examining armour) They are very strong and nice."
```

Comment:
I am leaving it to personal choice. I will leave anything I see as it is, anything new I write will likely be British English. I will try and keep it consistent with the current wording for any suggestions I put to you for For Power.